app/workers/worker_base.rb

```diff
@@ -0,0 +1,7 @@
+class WorkerBase
```

Comment:
if the only intention for introducing `WorkerBase` is to add the `perform_async_in_queue` behaviour, i'd suggest you open the `Sidekiq::Worker::ClassMethods` module instead:

```
module Sidekiq::Worker::ClassMethods
  def perform_async_in_queue(queue, *args)
    client_push('class' => self, 'args' => args, 'queue' => queue)
  end
end
```

this snippet could go into `config/initializers/sidekiq.rb`